src/garage/torch/algos/ppo.py

```diff
@@ -11,10 +11,11 @@ class PPO(VPG):
         env_spec (garage.envs.EnvSpec): Environment specification.
         policy (garage.torch.policies.base.Policy): Policy.
         baseline (garage.np.baselines.Baseline): The baseline.
-        optimizer (Union[type, tuple[type, dict]]): Type of optimizer.
+        optimizer (object): Type of optimizer.
```

Comment:
is there a reason you changed this type? was it inaccurate?

Reply:
No, I revert the code